contrib/fwupd.spec.in

```diff
@@ -82,7 +82,7 @@ BuildRequires: flashrom-devel >= 1.2-2
 %endif
 
 %if 0%{?have_modem_manager}
-BuildRequires: ModemManager-glib-devel >= 1.10.0
+BuildRequires: ModemManager-glib-devel >= 1.17.1
```

Comment:
Please don't explicitly bump these up unless it's really needed

Reply:
You'll notice CI failed because of it.  We need the build to keep working on older OSes.

```
error: Failed build dependencies:
	ModemManager-glib-devel >= 1.17.1 is needed by fwupd-1.6.1-0.1alpha.fc34.x86_64
```

Reply:
OK, I will keep libmm-glib >= 1.10.0 here.
Is there need me to add BuildRequires: libmbim-devel >= 1.25.3?

Reply:
>Is there need me to add BuildRequires: libmbim-devel >= 1.25.3?

It's new build dependency right?  I would suggest to make it "optional" dependency in meson.build (set a `HAVE_LIBMBIM` or similar) and  in this spec add `BuildRequires: libmbim-devel` without the version.

Reply:
> It's new build dependency right? I would suggest to make it "optional" dependency in meson.build (set a `HAVE_LIBMBIM` or similar) and in this spec add `BuildRequires: libmbim-devel` without the version.

Yes, it is a new build dependency, and I will make it optional in meson.build as you suggested, thanks a lot.